ml-agents/mlagents/trainers/curriculum.py

```diff
@@ -117,3 +111,27 @@ def get_config(self, lesson=None):
         for key in parameters:
             config[key] = parameters[key][lesson]
         return config
+
+    @staticmethod
+    def load_curriculum_file(location):
```

Comment:
I kept these in the same file, but I can move them to trainer_util.py if that's cleaner. It's currently the only place we load json.